babel.config.js

```diff
@@ -22,6 +22,7 @@ const defaultAlias = {
   '@mui/x-data-grid-generator': './packages/grid/x-grid-data-generator/src',
   '@mui/x-data-grid-pro': './packages/grid/x-grid/src',
   '@mui/x-license-pro': './packages/x-license/src',
+  'typescript-to-proptypes': '@material-ui/monorepo/packages/typescript-to-proptypes/src',
```

Comment:
Why have the absolute path here instead of a relative one like on the `docs` bellow ?

Reply:
I did this way solely to not break the line. Adding `../node_modules` makes it too long.